Aptoide/aptoide-client-v8
# [feature/ASV-1756] Wallet Install Dialog - API < 21 Error

**What does this PR do?**

   This implements a view for unsupported devices on wallet install dialog devices. This view should show up when a device has API < 21 only.

**Database changed?**

    No

**How should this be manually tested?**

  Verify that view is correct with the design and that this view only shows up if the device API < 21. Otherwise the regular flow applies.

**What are the relevant tickets?**

  [ASV-1756](https://aptoide.atlassian.net/browse/ASV-1756)

**Code Review Checklist**
